syntax = "proto3";

package brain.language;

message RawText {
  string text = 1;
}
